tests/contrib/openai/test_openai_v1.py

```diff
@@ -1346,3 +1346,114 @@ async def handle_async_request(self, request: httpx.Request) -> httpx.Response:
         assert False, f"Unexpected exception: {e}"
 
     assert asyncio_timeout, "Expected asyncio.TimeoutError"
+
+
+@pytest.mark.skipif(
+    parse_version(openai_module.version.VERSION) < (1, 66), reason="Response options only available openai >= 1.66"
+)
+@pytest.mark.snapshot(token="tests.contrib.openai.test_openai.test_response_completion")
+def test_response_completion(openai, mock_tracer):
+    """Ensure llmobs records are emitted for response endpoints when configured."""
+    with get_openai_vcr(subdirectory_name="v1").use_cassette("response_create.yaml"):
```

Comment:
```suggestion
    with get_openai_vcr(subdirectory_name="v1").use_cassette("response.yaml"):
```
Let's rename the cassette to be more concise